### PR TITLE
 Specific type description for participants and content done

### PR DESCRIPTION
**Pull request by Khanjan04**
Changed the type description for participants and content.

### CODE REVIEW

**Comment by Khanjan04** [comment]
Please have a look to these changes. And let me know if there is any suggestion.

File: src/components/challenges/active.tsx [hunk 1]
```diff
@@ -5,13 +5,26 @@ import Details from './details';
...
 type ActiveProps = {
-  content: {
-    is_user_subscribed: number;
+  content: {id: number;
     title: string;
     level: string;
     start_date: string;
     end_date: string;
-    participants: [];
+    participants: {user_id: string,
```
Comment (whyDontI) [inline src/components/challenges/active.tsx:13]: Can we move this `user_id` on new line?

File: src/components/challenges/active.tsx [hunk 2]
```diff
@@ -5,13 +5,26 @@ import Details from './details';
...
+      designation: string,
+      img: string,
+      github_id: string,
+      linkedin_id: string,
+      twitter_id: string,
+      instagram_id: string,
+      is_member: number,
+      rds_member_id: string}[];
+    is_active: number;
+    is_user_subscribed: number;
```
Comment (whyDontI) [inline src/components/challenges/active.tsx:27]: Can we fix the formatting?

**Review by whyDontI — CHANGES_REQUESTED** [review]
Can we fix formatting for all the types/interface definitions?

File: src/components/challenges/active.tsx (hunk 1 above)
Comment (Khanjan04) [reply src/components/challenges/active.tsx:13]: Sure. let me do that

File: src/components/challenges/active.tsx (hunk 2 above)
Comment (Khanjan04) [reply src/components/challenges/active.tsx:27]: Let me do that

**Comment by Khanjan04** [comment]
Have done required changes. Please have a look.

**Comment by Khanjan04** [comment]
Done.

**Comment by Khanjan04** [comment]
Done Nikhil. Check it once

**Review by whyDontI — APPROVED** [review]
Great work @Khanjan04 🥳 🔥

**Comment by Khanjan04** [comment]
Can I merge PR?

**Pull request merged**